docs/source/adapters/batchsystem.rst

```diff
@@ -0,0 +1,113 @@
+=====================
+Batch System Adapters
+=====================
+
+Fake Batch System Adapter
+-------------------------
+
+.. |FakeBatchSystemAdapter.get_utilization| replace:: :py:meth:`~tardis.adapters.batchsystems.fakebatchsystem.FakeBatchSystemAdapter.get_utilization`
+
+.. |FakeBatchSystemAdapter.get_allocation| replace:: :py:meth:`~tardis.adapters.batchsystems.fakebatchsystem.FakeBatchSystemAdapter.get_allocation`
+
+.. |FakeBatchSystemAdapter.get_machine_status| replace:: :py:meth:`~tardis.adapters.batchsystems.fakebatchsystem.FakeBatchSystemAdapter.get_machine_status`
+
+The :py:class:`~tardis.adapters.batchsystems.fakebatchsystem.FakeBatchSystemAdapter`
+implements a batch system adapter that mocks the response of hypothetical batch system.
+It can be used for testing purposes as well as a demonstrator in workshops and tutorials.
+
+The mocked response to the |FakeBatchSystemAdapter.get_allocation|, |FakeBatchSystemAdapter.get_utilization| and
+|FakeBatchSystemAdapter.get_machine_status| API calls is configurable statically in the adapter configuration.
+
+Available configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++----------------+---------------------------------------------------------------------+-----------------+
+| Option         | Short Description                                                   | Optionality     |
```

Comment:
*Optionality* is strange in this context. What do you think about *Requirement*?

```suggestion
| Option         | Short Description                                                   | Requirement     |
```